### PR TITLE
673 mini manage changes

### PR DESCRIPTION
**Pull request by beth-dixxon**
# Context
Updated design and content on mini-manage UI based off conversations we've had with Dan and Sarah. 

https://trello.com/c/QhHRnNQI

Miro board with updates: https://miro.com/app/board/uXjVPZfwSOY=/?share_link_id=248809937803

### CODE REVIEW

File: cypress_shared/pages/apply/releaseDate.ts [hunk 1]
```diff
@@ -2,6 +2,6 @@ import Page from '../page'
 
 export default class ReleaseDatePage extends Page {
   constructor() {
-    super('Do you know Robert Brown’s release date?')
+    super('Do you know the persons release date?')
```
Comment (pezholio) [inline cypress_shared/pages/apply/releaseDate.ts:5]: This has been updated now to include the person's actual name

**Comment by pezholio** [comment]
Sorry, just closed and reopened this PR to try and kick the build off again

**Comment by pezholio** [comment]
@beth-dixxon Lemme know if you need help with these conflicts

File: server/utils/bookingUtils.test.ts [hunk 2]
```diff
@@ -31,14 +31,14 @@ describe('bookingUtils', () => {
...
-              text: 'Extend booking',
+              text: 'Extend placement',
               classes: 'govuk-button--secondary',
               href: paths.bookings.extensions.new({ premisesId: 'premisesId', bookingId: booking.id }),
             },
             {
               classes: 'govuk-button--secondary',
               href: paths.bookings.cancellations.new({ premisesId: 'premisesId', bookingId: booking.id }),
-              text: 'Cancel booking',
+              text: 'Cancel placemeent',
```
Comment (pezholio) [inline server/utils/bookingUtils.test.ts:41]: There's a typo here 😅

**Review by richpjames — APPROVED** [review]

**Pull request merged**